themes/bootstrap3/templates/ajax/resolverLink.phtml

```diff
@@ -0,0 +1,22 @@
+<?php if (!empty($link['href'])): ?>
+  <a href="<?=$this->escapeHtmlAttr($link['href'])?>" title="<?=isset($link['service_type']) ? $this->transEsc($link['service_type']) : ''?>"<?=!empty($link['access']) ? ' class="access-' . $link['access'] . '"':''?>>
+    <?=isset($link['title']) ? $this->escapeHtml($link['title']) : ''?>
```

Comment:
This could probably simplify down to `<?=$this->escapeHtml($link['title'] ?? '')?>` both here and below if we're trying to make this as concise as possible.

Reply:
Done!